wxt_cidrbot/Git_handler.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import sys
+from github import Github
+
+
+class githandler:
+    def __init__(self):
+        logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+        self.logging = logging.getLogger()
+
+        # Connect to git, right now using no tokens
+        self.git_init = Github()
```

Comment:
a better name for `git_init` would be something like `git_api_handle` or `git_api` -